CMS-Enterprise/easi-app
# Easi 4470/dataloaders linked

# EASI-4470

## Description


- Hook up linked TRB Requests and linked System Intakes to dataloaders

## How to test this change

- pull branch and create new system intake locally
- link it to existing systems, however many you want
- open postman, and navigate to `Cedar/Core/System Summary/Get System`
- use the `id` of the cedar system you linked the system intake to
- confirm the response contains the system intake id in the `linkedSystemIntakes` field
- you can create any number of TRB requests and system intakes and add the cedar system as the system
- resend the postman request and confirm the ids all show up where they should under `linkedSystemIntakes` and `linkedTrbRequests`

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
